ubuntudesign/vanilla-framework
# Rename p-image-grid to p-inline-images

## Done

- Rename p-image-grid to p-inline-images
- Tidy up docs
- Edit example to show different width images

## QA

- Check code
- Check local works as demo - https://vf-io-7e29b.firebaseapp.com/patterns/inline-images/

## Details

Fixes #615, #586
